compat.c

```diff
@@ -469,7 +472,7 @@ char	*strsep(char **string_p, const char *delim)
  * Local getenv which handles some portability stuff.
  */
 char	*loc_getenv(const char *var, char *buf, const int buf_size,
-		    const int stay_safe)
+		    const int stay_safe, const int copy_to_buf)
```

Comment:
why need add copy_to_buf? it isn't too harmful to copy env variable every time, since loc_getenv is never on the critical performance path.

Reply:
I think the new update from https://github.com/j256/dmalloc/pull/58 is better than this.